components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -21,29 +21,42 @@ export interface ProjectCheckedMap {
   templateUrl: './projects-dropdown.component.html',
   styleUrls: ['./projects-dropdown.component.scss']
 })
-export class ProjectsDropdownComponent implements OnChanges {
+export class ProjectsDropdownComponent implements OnInit, OnChanges {
   // The map of ProjectChecked by id. Any checked changes propagated via
   // onProjectChecked. Updates should be applied to parent component state.
   @Input() projects: ProjectCheckedMap = {};
 
   // Setting disabled to true means the dropdown will be unusable and will have a grey background
   @Input() disabled = false;
 
+  // Used to re-synchronize summary label if the set of checked items has changed.
+  // This optional input is needed only when re-displaying the project dropdown
+  // for *additional* resources, as with the create-object-modal-component.
+  // Other consumers, e.g. team-details.component use it only for a single resource.
+  @Input() projectsUpdated: EventEmitter<boolean>;
+
   // Emits a project that changed as a result of a check or uncheck.
   @Output() onProjectChecked = new EventEmitter<ProjectChecked>();
 
-  active = false;
-  label = UNASSIGNED_PROJECT_ID;
+  public active = false;
+  public label = UNASSIGNED_PROJECT_ID;
 
-  projectsArray(): ProjectChecked[] {
-    const projects = Object.values(this.projects);
-    return ChefSorters.naturalSort(projects, 'name');
+  ngOnInit(): void {
+    if (this.projectsUpdated) { // an optional setting
+      this.projectsUpdated.subscribe(() => {
+        this.updateLabel();
+      });
+    }
```

Comment:
This is the other half of the fix: resyncing the label to the now all-unchecked list of projects.